spec/helpers/formatters.spec.js

```diff
@@ -172,7 +172,7 @@ describe("formatters", function() {
 		});
 
 		it("Doesn\"t override any styles", function() {
-			expect(initStyles(this.defaultStyles, {})).toEqual(this.defaultStyles);
+			expect(initStyles(this.defaultStyles)).toEqual(this.defaultStyles);
```

Comment:
Seems like a superfluous test now

Reply:
Probably for code coverage, I will fix that " though